docs/learn-more/faq.md

```diff
@@ -43,35 +42,16 @@ When there is a new release of **source{d} CE**, it is noticed every time a `sou
 3. run `sourced compose download`
 4. run `sourced restart` to apply the new configuration.
 
-This process will reinstall **source{d} CE** with the new components, but it will keep your data \(repositories, charts, dashboards, etc\). If you want to replace all the current dashboards with the ones from the release that you just installed, you have two alternatives:
+This process will reinstall **source{d} CE** with the new components, but it will
+keep your data (repositories, charts, dashboards, etc).
 
-* run `sourced prune --all` before running `sourced init`; but if you were using
+If you want to replace all the current dashboards with the ones from the release
+that you just installed, the official way to proceed is to run `sourced prune --all` before running `sourced init`; but it will also delete your saved queries and charts,
+and if you were using repositories downloaded from a GitHub organization, they
+will be deleted, and downloaded again.
 
-  repositories downloaded from a GitHub organization, they will be deleted, and
-
-  downloaded again.
-
-* drop only the dashboards, and load the new ones, following also this other
-
-  instructions: [How To Restore Dashboards and Charts to Defaults](faq.md#how-to-restore-dashboards-and-charts-to-defaults)
-
-## How To Restore Dashboards and Charts to Defaults
-
-In some circumstances, you may want to restore the UI dashboard and charts to its defaults. Currently, there is no clean way of doing it without deleting all the state of the UI.
-
-Following these steps, all the state modified from the UI \(charts, dashboards, saved queries, users, roles, etcetera\) will be replaced by the default ones for the version of **source{d} CE** that you're currently using. If you're using repositories from a GitHub organization, all its data will be preserved, and only charts and dashboards will be restarted.
-
-To do so, you only need to delete the docker volume containing the PostgreSQL database, and restart **source{d} CE**. It can be done following these steps if you already have [Docker Compose](https://docs.docker.com/compose/) installed:
-
-```text
-$ cd ~/.sourced/workdirs/__active__
-$ source .env
-$ docker-compose stop postgres
-$ docker-compose rm -f postgres
-$ ENV_PREFIX=`awk '{print tolower($0)}' <<< ${COMPOSE_PROJECT_NAME}`
-$ docker volume rm ${ENV_PREFIX}_postgres
-$ sourced restart
-```
+If you only want to restore the dashboards, loading the ones provided by the version
```

Comment:
I'm still not so sure about redirecting end users to the docs for developers. A user can do a `prune` + `init`, that should be enough for the FAQ.

Reply:
not really agree, but let's do as you suggest. Link removed.

Reply:
thanks